requirements-server.txt

```diff
@@ -1,4 +1,5 @@
 area-detector-handlers
+bson
```

Comment:
I think this is probably not what is wanted.

The `pymongo` pip package provides a library named `bson` but it's not the same as what `pip install bson` provides. If this was placed here in response to some `ImportError` it probably indicates that `pymongo` was not installed properly.